sysboundary/ionosphere.h

```diff
@@ -37,6 +37,7 @@ using namespace std;
 
 namespace SBC {
 
+   constexpr static int productionNumParticleEnergies = 100;
```

Comment:
Why this change? It isn't in dev. And this shouldn't affect eVlasiator?

Reply:
It is here, PR#666. Some compilers complain about const-ness of the grid object that used to contain this int, or something to that effect.
https://github.com/fmihpc/vlasiator/blob/04fda55acf74fcd075c9a92fa94b769ffc6cd578/sysboundary/ionosphere.h#L45

Reply:
Ah, well, in dev all of those constexpr parameters have been moved up to this location. Cherry-pick the relevant commit? 5ee8c83bd7f88ff918b22ec3c2947cc90ced395d